_posts/blog/2023-07-01-2023-retreat-okrs.md

```diff
@@ -2,47 +2,62 @@
+<p class="text-center"><img src="/images/blog/2023-board-retreat.jpg" alt="Picture of our Board of Directions from our board retreat" class='img-responsive' style='width:100%;'/>
+</p>
+
+On June 3th, 2023, the Board of Directors held our first in-person *board retreat*! This retreat served as an excellent opportunity for the 10 Board of Directors to have more focused discussions around what membership means at Chi Hack Night, plan our upcoming **OKRs** for the year, and enjoy copious amounts of bagels & coffee. 
 
 Our retreat was held from 10 am until 3 pm, downtown Chicago at [dscout](www.dscout.com) corporate headquarters. (Thank you [dscout](www.dscout.com) for graciously hosting us for the day!)
 
-# Membership Discussion
+In planning our goals for the upcoming year - three larger goals out of our discussions: 
+1. Continuing to run a high quality civic tech event
+2. Align our membership experiences with our hybrid event
+3. Expand the broader awareness of Chi Hack Night
+
+For the third year, we've used an **OKR** framework to plan and measure our goals. **OKRs** (short for **Objective** & **Key Results**) help us know: 
+1. What are our important focus areas - the **Objectives** 
+2. How do we know we've succeeded - the **Key Results**
+
+*If you are curious to learn more about OKRs and how they work, here's an [article](https://www.whatmatters.com/faqs/okr-meaning-definition-example) that shares more about how they work.*
 
-# OKR Planning
+While you'll see and hear more about the specifics on various ways of going about this, we wanted to share these out so that our wider community can see what we've determined to focus on for the coming year. 
 
-## 2023-2024 OKRs
+# **2023-2024 OKRs**
 *from 7/1/23 - 6/30/24*
 
-### **Objective 1 - Run the event** {#objective-1-run-the-event}
+## **Objective 1 - Run the event** {#objective-1-run-the-event}
 _Produce a live, weekly civic engagement event that meaningfully includes diverse speakers and attendees_
-* **KR 1** - 
-* **KR 2** - 
-* **KR 3** - 
+* **KR 1** - Create a biannual survey for Chi Hack Night community asking people about belonging / welcomingness of events & quality and representativeness of presenters
+* **KR 2** - Book at least three presentations from each of the following groups: government, journalism, academia, nonprofit, and advocacy
+* **KR 3** - 65% or higher response rate from presenters on our demographic survey.  
 
-### **Objective 2 - Increase Membership Engagement** {#objective-2-increase-membership-engagement}
-* **KR 1** - X% of volunteer roles available (non-board required) are filled by members (either current or future members)
+
+## **Objective 2 - Increase Membership Engagement** {#objective-2-increase-membership-engagement}
+* **KR 1** - 20% of volunteer roles available are filled by members (either current or future members)
```

Comment:
Is this non-board member members?

Reply:
yes - non-board members. I intentionally did not include that piece for brevity - cuz then I felt like I would need to explain why that stipulation is needed. So I just opted to keep it short.